autode/species/complex.py

```diff
@@ -252,45 +316,75 @@ def __init__(self, *args, name='complex', do_init_translation=False):
 
         Keyword Arguments:
             name (str):
-            do_init_translation (bool): Translate molecules initially such
-                                        that they do not overlap
-        """
-        self.molecules = args
-        self.molecule_atom_indexes = []
 
-        # Calculate the overall charge and spin multiplicity on the system and
-        # initialise
-        complex_charge = sum([mol.charge for mol in self.molecules])
-        complex_mult = (sum([mol.mult for mol in self.molecules])
-                        - (self.n_molecules - 1))
+            do_init_translation (bool): Translate molecules initially such
+                                        that they donot overlap
 
-        complex_atoms = []
-        for mol in self.molecules:
-            complex_atoms += deepcopy(mol.atoms)
+            copy (bool): Should the molecules be copied into this complex?
 
+            solvent_name (str | None): Name of the solvent, if None then select
+                                       the first solvent from the constituent
+                                       molecules
+        """
         super().__init__(name=name,
-                         atoms=complex_atoms,
-                         charge=complex_charge,
-                         mult=complex_mult)
+                         atoms=sum((deepcopy(mol.atoms) if copy else mol.atoms
```

Comment:
what does the "sum" do here?

Reply:
just adds the atoms together. shorthand for 

```
atoms = []
for mol in args:
    atoms += mol.atoms 
```